### PR TITLE
docs(contributing): document test types

### PR DESCRIPTION
**Pull request by adamweeks**
## Description

Documentation update for test types.

## Type of Change

[x] Documentation

## Checklist:

- [X ] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] I have made corresponding changes to the documentation
- [ X] My changes generate no new warnings
- [ X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: karma-ng.conf.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ function makeConfig(packageName, argv) {
   /* eslint complexity: [0] */
   const launchers = makeBrowsers(packageName, argv);
   const integrationTestPath = path.join('packages', 'node_modules', packageName, 'test', 'integration', 'spec', '**', '*.js');
-  const unitTestPath = path.join('packages', 'node_modules', packageName, 'test', 'unit', 'spec', '**', '*.js');
+  const unitTestPath = path.join('packages', 'node_modules', packageName, 'src', '**', '*-spec.js');
+  const legacyUnitTestPath = path.join('packages', 'node_modules', packageName, 'test', 'unit', 'spec', '**', '*.js');
```
Comment (mattnorris) [inline karma-ng.conf.js]: @adamweeks is this change still necessary/helpful? 
@taymoork2 will this change still jive with our tests?

Comment (adamweeks) [reply karma-ng.conf.js]: @mattnorris no clue to be honest. I think we had originally wanted to change the unit test files to `-spec.js`, but I don't see the need for it now. Probably can close this and maybe take some of the doc changes into a new PR
